package.json

```diff
@@ -60,7 +61,7 @@
     "webpack-cli": "^3.3.10"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=12.0.0"
```

Comment:
is this supposed to be 13?

Reply:
Local development would easily fail to run all the tests without memory failures on Node v10. Node v12's improved memory management allowed tests to run consistently without memory issues. This setting ensures developers to use at least v12 in order to develop effectively. We use Node v13 on CI to take advantage of any more recent Node performance/memory improvements.